README.md

```diff
@@ -20,6 +20,66 @@ Fork this repository and add your tables here:
 
 (your tables go here - you can use TableGenerator to create them. Include the data from the sample above in your tables)
 
+### Authors
+
+| AuthorID | FullName      | Email                       | Bio            |
+|----------|---------------|-----------------------------|----------------|
+| 1        | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here |
+| 2        | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here |
+| 3        | Steve Smith   | steve@nimblepros.com        | Long Text Here |
+
+### BlogPosts
+
+| Slug                          | Title                                | Body           | AuthorID | Views |
+|-------------------------------|--------------------------------------|----------------|----------|-------|
+| serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here | 1        | 123   |
+| persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here | 2        | 234   |
+| localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here | 3        | 345   |
+
+### Categories
+
+| CategoryID | Category             |
+|------------|----------------------|
+| 1          | Software Development |
+| 2          | Cloud Computing      |
+| 3          | Web APIs             |
+
+### Tags
+
+| TagID | Tag          |
+|-------|--------------|
+| 1     | logging      |
+| 2     | serilog      |
+| 3     | appinsights  |
+| 4     | smartenum    |
+| 5     | efcore       |
+| 6     | localization |
+| 7     | aspnetcore   |
+| 8     | web-api      |
+
+### SlugCategories
+
+| Slug                          | CategoryID |
+|-------------------------------|------------|
+| serilog-structured-logging    | 1          |
+| serilog-structured-logging    | 2          |
+| persisting-smart-enum         | 1          |
+| localization-aspnet-core-apis | 1          |
+| localization-aspnet-core-apis | 3          |
```

Comment:
I see, you're joining on the slug instead of the Id. That's fine and will work but more typically you'd join on two Ids, like... oh right you don't have a BlogPost.Id column. But if you did, I'd use that. :)

Reply:
Yes. You can even eliminate the Id column usually. The two PKs are sufficient and should serve as a composite key.

However, some ORMs work better if there is an ID column, and sometimes you want to put more metadata on here which again starts to turn it into a "real" entity instead of just a relationship table. For example, CreatedDate, CreatedBy, etc.